pkgs/by-name/ar/arma3-unix-launcher/package.nix

```
@@ -0,0 +1,85 @@
+{ lib
+, stdenv
+, cmake
+, curl
+, curlpp
+, doctest
+, fetchFromGitHub
+, fetchurl
+, fmt
+, nlohmann_json
+, qt5
+, spdlog
+, substituteAll
+, trompeloeil
+, buildDayZLauncher ? false
+,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "arma3-unix-launcher";
+  version = "383-unstable-2024-07-05";
+  src = fetchFromGitHub {
+    owner = "muttleyxd";
+    repo = finalAttrs.pname;
+    rev = "f77f9e4c759b32718a589b282a4146fa0b930242";
+    hash = "sha256-0cU3pirDx+hwMqYBEvwlSpdAgJDhnC1m3+mgZ7YP49Y=";
+  };
+  nativeBuildInputs = [
+    qt5.wrapQtAppsHook
+    cmake
+    spdlog
+    curlpp.src
+    curl
+  ];
+
+  buildInputs = [
+    qt5.qtbase
+    qt5.qtsvg
+  ];
+
+  cmakeFlags = [ "-Wno-dev" ] ++ lib.optional buildDayZLauncher [ "-DBUILD_DAYZ_LAUNCHER=ON" ];
```

```suggestion
  cmakeFlags = [ "-Wno-dev" ] ++ lib.optionals buildDayZLauncher [ "-DBUILD_DAYZ_LAUNCHER=ON" ];
```

`optional` would make it a nested list

---

actually since we have `lib.cmakeBool`, the latter could also be `lib.cmakeBool "BUILD_DAYZ_LAUNCHER" buildDayZLauncher`